client/lib/ads/utils.js

```diff
@@ -11,16 +10,15 @@ import { isBusiness, isPremium } from 'lib/products-values';
  * @return {boolean}      true if site has WordAds access
  */
 export function canAccessWordads( site ) {
-	if ( site && config.isEnabled( 'manage/ads' ) ) {
+	if ( site ) {
 		if ( isWordadsInstantActivationEligible( site ) ) {
 			return true;
 		}
 
 		const jetpackPremium = site.jetpack && ( isPremium( site.plan ) || isBusiness( site.plan ) );
 		return site.options &&
 			( site.options.wordads || jetpackPremium ) &&
-			userCan( 'manage_options', site ) &&
-			( ! site.jetpack || config.isEnabled( 'manage/ads/jetpack' ) );
```

Comment:
Yeah, Jetpack is a first-class citizen now. Though `userCan( 'manage_options', site )` needs to stay,